src/test/java/org/assertj/core/api/abstract_/AbstractAssert_isInstanceOfSatisfying_Test.java

```diff
@@ -48,6 +48,16 @@ protected ConcreteAssert create_assertions() {
     return new ConcreteAssert(yoda);
   }
 
+  @Override
+  protected ConcreteAssert applyIgnoreWithoutEffects() {
+    return assertions.ignoreWhen(v -> ((Jedi) v).getName().equals("Yoda"));
+  }
```

Comment:
I believe we don't need to override this if we used `v -> true` for `applyIgnoreWithoutEffects` implementation

Reply:
You are completely right, it would make it way easier. I'll try with that